Student_t.ipynb

```diff
@@ -0,0 +1,1086 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def studentt(range, number, save_name, fig=None, ax=None):
</pre><p><code>range</code> is python's keyword, I guess we should not use as a argument to a function.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>